jwst/assign_wcs/nirspec.py

```diff
@@ -1348,7 +1368,7 @@ def get_slit_location_model(slitdata):
     return model
 
 
-def gwa_to_ymsa(msa2gwa_model, lam_cen=None, slit=None):
+def gwa_to_ymsa(msa2gwa_model, lam_cen=None, slit=None, slit_y_range=None):
```

Comment:
Is this a safe default value for `slit_y_range`?

Reply:
`slit_y_range` is used only in IFU mode and it should always be populated. I can't make it a position argument because it's only for IFU (as `slit` is for MOS and FS only). But I want it to fail if the value was not passed.